packages/p2p/webpack.config.js

```diff
@@ -0,0 +1,90 @@
+const path = require('path');
+
+const is_serve   = process.env.BUILD_MODE === 'serve';
+const is_release = process.env.NODE_ENV === 'production' || process.env.NODE_ENV === 'staging';
+
+module.exports = {
+    entry: {
+        'index'  : path.resolve(__dirname, 'src/components', 'app.jsx'),
+    },
+    output: {
+        path         : path.resolve(__dirname, 'lib'),
+        filename     : '[name].js',
+        libraryExport: 'default',
+        library      : ['deriv-p2p', '[name]'],
+        libraryTarget: 'umd',
+    },
+    module : {
+        rules: [
+            (!is_serve ? {
+                enforce: 'pre',
+                test   : /\.(js|jsx)$/,
+                exclude: /node_modules/,
+                include: /src/,
+                loader : 'eslint-loader',
+                options: {
+                    fix: true,
+                },
+            } : {}),
+            {
+                test   : /\.(js|jsx)$/,
+                exclude: /node_modules/,
+                use    : [
+                    {
+                        loader: 'deriv-shared/utils/deriv-components-loader.js'
+                    },
+                    {
+                        loader : 'babel-loader',
+                        options: {
+                            presets: [
+                                '@babel/preset-env',
+                                '@babel/preset-react'
+                            ],
+                            plugins: [
+                                ["@babel/plugin-proposal-decorators", { "legacy": true }],
```

Comment:
The `options` specified should not be necessary, if I'm not mistaken, since we have a `.babelrc` file in package root. Can we try this?

Reply:
yep, you're right